bluebutton-data-pipeline-rif-extract/src/test/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/s3/task/ManifestEntryDownloadTaskIT.java

```diff
@@ -0,0 +1,78 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.extract.s3.task;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.transfer.Download;
+import com.justdavis.karl.misc.exceptions.BadCodeMonkeyException;
+
+import gov.hhs.cms.bluebutton.datapipeline.rif.extract.ExtractionOptions;
+import gov.hhs.cms.bluebutton.datapipeline.rif.extract.exceptions.AwsFailureException;
+import gov.hhs.cms.bluebutton.datapipeline.rif.extract.exceptions.ChecksumException;
+
+/**
+ * Tests downloaded S3 file attributes such as MD5ChkSum
+ */
+public final class ManifestEntryDownloadTaskIT {
+	private static final Logger LOGGER = LoggerFactory.getLogger(ManifestEntryDownloadTask.class);
+
+	private S3TaskManager s3TaskManager;
+
+	public ManifestEntryDownloadTaskIT() {
+		this.s3TaskManager = null;
+
+	}
+
+	/**
+	 * Test to ensure the MD5ChkSum of the downloaded S3 file matches the generated
+	 * MD5ChkSum value
+	 */
+	@Test
+	public void testMD5ChkSum()
```

Comment:
This test should be reworked, as it doesn't inherently need to rely on already-existing S3 resources (and definitely shouldn't rely on that particular S3 bucket). Instead, I'd do the following:

1. Create a new, random S3 bucket (see other ITs for this).
2. Upload your small test file to that bucket.
3. Run your test.
4. Delete the random S3 bucket.